### ISSUE TITLE
Clamp slice ends to known tensor size

### ISSUE THREAD

**Issue by nlgranger** [daquexian/onnx-simplifier#120]
For unspecified slice ranges such as:
```py
a = a[:, 3]
```
Pytorch onnx export generates slices with ends attribute set to the maximum int64 value (9223372036854775807).
This causes tensorRT to spit out warnings every time it clamps this value to int32 range.
Would it be possible to clamp ends to the tensor size when its size is known?

**Comment by daquexian** [comment]
Thanks for your issue! It is possible to clamp it. But in my opinion it is just a warning so clamp it or not is not so important? Please correct me if I'm wrong

**Comment by nlgranger** [comment]
Indeed, the warning is harmless in most of the reasonable use-cases. This feature request is for convenience only.
